tests/tasks/git-add-commit-push.yaml

```diff
@@ -14,6 +14,12 @@
     dest: /{{git_repo_name}}/
   when: docker_files is defined
 
+- name: "{{ testname }} - removing cronjobs from .lagoon.yml"
+  copy:
+    src: /{{git_repo_name}}/{{lagoon_no_crons}}
+    dest: /{{git_repo_name}}/.lagoon.yml
+  when: lagoon_no_crons is defined
```

Comment:
let's call this variable `overwrite_lagoon_yml` as then it will be much more general and can be used by all kind of tests to overwrite the `.lagoon.yml` and not only for crons

Reply:
Resolved in c40d7205f6d88c081ac99c50eb965ae789810322